views/nexteuropa_newsroom_views_handler_field_taxonomy.inc

```diff
@@ -10,6 +10,8 @@
  *
  * @ingroup views_field_handlers
  */
+
+// @codingStandardsIgnoreLine
```

Comment:
you even can ignore the whole file in the ruleset, because it's a views handler.